2018-04-03T15:28:31.869Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: error executing chaincode: transaction returned with failure: [composerchannel-a2c73b49]Calling chaincode Invoke() returned error response [Error: Failed to add object with ID 'NaN' in collection with ID 'Participant:org.gov.fundtracker.FundTransferApprover' as the object already exists]. Sending ERROR message back to peer
    at /home/vagrant/hyperledger/server/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-04-03T15:28:31.871Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: error executing chaincode: transaction returned with failure: [composerchannel-a2c73b49]Calling chaincode Invoke() returned error response [Error: Failed to add object with ID 'NaN' in collection with ID 'Participant:org.gov.fundtracker.FundTransferApprover' as the object already exists]. Sending ERROR message back to peer\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: error executing chaincode: transaction returned with failure: [composerchannel-a2c73b49]Calling chaincode Invoke() returned error response [Error: Failed to add object with ID 'NaN' in collection with ID 'Participant:org.gov.fundtracker.FundTransferApprover' as the object already exists]. Sending ERROR message back to peer\n\n    at _initializeChannel.then.then.then.then.catch (/home/vagrant/hyperledger/server/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:927:34)\n    at <anonymous>"}$
